CMakeLists.txt

```diff
@@ -76,6 +77,15 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
+if(ENABLE_THREADS)
+    if(CMAKE_USE_PTHREADS_INIT)
+        message(STATUS "Threads type: pthread")
+    else()
+        message(STATUS "Threads type: Win32 thread")
+    endif()
```

Comment:
```cmake
message(STATUS "Threads type: ${CMAKE_THREAD_LIBS_INIT}")
```

Reply:
Just tested it. CMAKE_THREAD_LIBS_INIT was empty on my mac

Reply:
Ah that may be because pthread is part of libc there.

Reply:
Let's keep it as it is.